<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTableDialog</class>
 <widget class="QDialog" name="NewTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Table</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>rows:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinR">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>columns:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinC">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
       <item>
        <widget class="QCheckBox" name="chkH">
         <property name="text">
          <string>header,  background color:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ColorComboBox" name="cmbHCol"/>
       </item>
       <item>
        <widget class="QCheckBox" name="chkHBold">
         <property name="text">
          <string>bold</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
       <item>
        <widget class="QCheckBox" name="chkBack">
         <property name="text">
          <string>background color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ColorComboBox" name="cmbBack">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>table:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="chkCentered">
       <property name="text">
        <string>centered</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>spacing:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinSpacing">
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>     padding:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinPadding">
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>     border:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBorder">
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Preview:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scroll">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>563</width>
        <height>264</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>colorcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewTableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewTableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkH</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbHCol</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBack</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbBack</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkH</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkHBold</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NewTableDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinC</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NewTableDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkH</sender>
   <signal>toggled(bool)</signal>
   <receiver>NewTableDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbHCol</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>NewTableDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkHBold</sender>
   <signal>toggled(bool)</signal>
   <receiver>NewTableDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBack</sender>
   <signal>toggled(bool)</signal>
   <receiver>NewTableDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbBack</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>NewTableDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkCentered</sender>
   <signal>toggled(bool)</signal>
   <receiver>NewTableDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinSpacing</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NewTableDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinPadding</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NewTableDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBorder</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NewTableDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>preview()</slot>
 </slots>
</ui>
